matplotcheck/base.py

```diff
@@ -1088,14 +1088,17 @@ def assert_line(
         AssertionError
             with message `m` or `m2` if no line exists that covers the dataset
         """
-        flag_exist, flag_length = False, False
+        flag_exist = False
+        flag_length = False
         xy = self.get_xy(points_only=True)
         min_val, max_val = min(xy["x"]), max(xy["x"])
 
         for l in self.ax.lines:
-            path_verts = self.ax.transData.inverted().transform(
-                l._transformed_path.get_fully_transformed_path().vertices
-            )
+            # Here we will get the verticies for the line and reformat them in
```

Comment:
lets:
1. Remove xtime as a parameter
2. Let's make the data extent (boolean) parameter to make this test OPTIONAL